app/client.go

```diff
@@ -98,7 +98,7 @@ func parseQuery(el *onet.Roster, sum string, count bool, where, predicate, group
 	whereRegex := "{(w[0-9]+(,\\s*[0-9]+))*(,\\s*w[0-9]+(,\\s*[0-9]+))*}"
 	groupByRegex := "{g[0-9]+(,\\s*g[0-9]+)*}"
 
-	if !checkRegex(sum, sumRegex) {
+	if checkRegex(sum, sumRegex) == false {
```

Comment:
? I thought @tharvik 's suggestion was to change from ` == false` to `!`, no? Anyway, that would be my preference.

Reply:
you're right I didn't see it correctly :(

Reply:
done fixed e5509ba